### ISSUE TITLE
Importing FMT_BINARY from plistlib yields no-name-in-module 

### ISSUE THREAD

**Issue by postlund** [PyCQA/pylint#3348]
### Steps to reproduce
Import and use `FMT_BINARY` form `plistlib`, e.g.:

```python
"""Open and print plist file."""
import sys
from plistlib import load, FMT_BINARY

with open(sys.argv[1], 'rb') as plist:
    print(load(plist, fmt=FMT_BINARY))
```

### Current behavior
pylint reports that `FMT_BINARY` is missing:

```shell
$ pylint plist.py
************* Module plist
plist.py:3:0: E0611: No name 'FMT_BINARY' in module 'plistlib' (no-name-in-module)
```

### Expected behavior
This constant exists so no error should be reported.

### pylint --version output

```shell
pylint 2.4.4
astroid 2.3.3
Python 3.8.1 (default, Jan  8 2020, 13:13:59)
[GCC 4.9.2]
```

**Comment by RhetTbull** [comment]
I arrived here after seeing the same issue but I think this is actually a plistlib issue, not a pylint issue.  plistlib creates FMT_BINARY (and FMT_XML) and inserts it into the module namespace at load:

https://github.com/python/cpython/blob/0d860dd43c72dc7046a5d18fc72d495cadd4a2df/Lib/plistlib.py#L68-L69
